src/api/controllers/TeamController.js

```diff
@@ -120,6 +121,78 @@ module.exports = {
       });
   },
 
+  getAll: (_, res) => {
+    var response = [];
+
+    Team.find()
+      .sort({ dateCreated: -1 })
+      .populate("categories")
+      .then((teams) => {
+        teams.forEach((team) => {
+          toSanitize = ["__v", "followers", "dateCreated", "owner"];
+          team = sanitize(team, toSanitize);
+          team.categories = team.categories.map((c) => c.name);
+
+          response.push(team);
+        });
+      })
+      .then(() => {
+        return res.send(response);
+      });
+  },
+
+  getTeam: (req, res) => {
+    Team.findOne({ _id: req.params.team_id })
+      .populate("categories")
+      .then((team) => {
+        if (team) {
+          toSanitize = ["__v", "owner", "dateCreated"];
```

Comment:
We need `dateCreated` in the frontend as well so let's keep it 😃 

```suggestion
          toSanitize = ["__v", "owner"];
```